pkg/kine/prepared/db.go

```diff
@@ -22,15 +35,27 @@ func New(db *sql.DB) *DB {
 
 func (db *DB) Underlying() *sql.DB { return db.underlying }
 
-func (db *DB) ExecContext(ctx context.Context, query string, args ...any) (sql.Result, error) {
+func (db *DB) ExecContext(ctx context.Context, query string, args ...any) (result sql.Result, err error) {
```

Comment:
It seems like you are not using your named returns. Perhaps you can modify this to anonymous results again:
```suggestion
func (db *DB) ExecContext(ctx context.Context, query string, args ...any) (sql.Result, error) {
```

Reply:
Keeping named returns as they are used in `span.RecordError(err)`